samples/excel/89-preview-apis/chart-series.yaml

```diff
@@ -0,0 +1,171 @@
+order: 1
+id: excel-89-preview-apis-chart-series-
+name: Chart series
+description: 'Add, set, and delete a series in a chart.'
+host: EXCEL
+api_set:
+    ExcelAPI: 1.7
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+        $("#add-series").click(() => tryCatch(addSeries));
+        $("#delete-series").click(() => tryCatch(deleteSeries));
+        $("#add-series-value").click(() => tryCatch(setSeriesValue));
+
+        async function addSeries() {
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getItem("Sample");
+
+                let seriesCollection = sheet.charts.getItemAt(0);
+                let rangeSelection = sheet.getRange("B1:B7");
+
+                // Add a series.
+                let newSeries = seriesCollection.series.add("New Series");
```

Comment:
Suggestion:  name the series after the column of data that you're adding as a series on the chart (since the series name is displayed in the chart legend).

- change this line: let newSeries = seriesCollection.series.add("New Series");
- to this instead: let newSeries = seriesCollection.series.add("Qtr2");

Reply:
Done